src/app/public/plugin-resources/code-examples/timepicker/basic/timepicker-demo.component.html

```diff
@@ -0,0 +1,42 @@
+<form
+  novalidate
+  [formGroup]="formGroup"
+>
+  <sky-timepicker
+    #timepicker
+  >
+    <input
+      formControlName="time"
+      type="text"
+      [skyTimepickerInput]="timepicker"
+    />
+  </sky-timepicker>
+
+  <p *ngIf="timeControl.errors && (timeControl.dirty || timeControl.touched)"
+    class="sky-error-label"
+  >
+    <span [hidden]="timeControl.errors && !timeControl.invalid">
+      Enter a valid time.
+    </span>
+
+    <span *ngIf="timeControl.errors.required">
+      Time is required. Select a value.
```

Comment:
I can't get the demo to display any of the error messages. This is probably OK, but just checking. If I enter invalid properties in the field and tab away, it is highlighted with a red border, but no error message appears. Is that the intended behavior?

Reply:
Yeah, I didn't add any validation errors to the demo for simplicity's sake. Those are accounted for in the Stackblitz _code examples_ however. Are you seeing the appropriate styles and error messages for the code examples?

Reply:
When I play around with it in Stackblitz and enter enter invalid values or no value, I get a combination of the first message and one of two messages that follow. So either: `Enter a valid time. Time is required. Select a value.` or `Enter a valid time. The time is invalid. Enter a valid time or use the picker.` Is that combination the intended behavior? It seems overly wordy to me. I think we can tighten the two error messages down to just `Time is required.` and `Enter a valid time.` I think that follows our guidance at https://developer.blackbaud.com/skyux/components/error#content.